applications/FluidDynamicsApplication/tests/manufactured_solution_test/manufactured_solution_ref4.mdpa

```diff
@@ -6729,7 +6729,7 @@ Begin Elements Element2D3N// GUI group identifier: Fluid
        156          0          8         13         10
        157          0          2          4          6
        158          0          4          8          6
-       159          0          0          3          2
```

Comment:
Was this a bug in mdpa? Zero index nodes are not allowed ryt? How come this mdpa worked before?

Reply:
Because the manufactured solution test is run with the three first meshes by default. I discovered this while switching on the finest mesh to check that everything was correct.